helm/release.sh

```diff
@@ -4,6 +4,9 @@ set -eauo pipefail
 
 TAG=${1:-$(git describe --abbrev=0 --tags)}
 
+HELM_GIT_REPO_URL="https://azure.github.io/application-gateway-kubernetes-ingress/helm"
+HELM_REPO_URL=${2:-$HELM_GIT_REPO_URL}
```

Comment:
now that we have more than 1 cli param we should perhaps add `usage` to this shell script - in the form of a comment  on the second line?